doc/controls/ResponsiveView.md

```diff
@@ -0,0 +1,131 @@
+---
+uid: Toolkit.Controls.ResponsiveView
+---
+# ResponsiveView
+
+## Summary
+
+The `ResponsiveView` provides the ability to display different content based on screen size, making it easy to adapt to various devices.
+
+### C#
+```csharp
+public partial class ResponsiveView : ContentControl
+```
+
+### XAML
+```xml
+xmlns:utu="using:Uno.Toolkit.UI"
+...
+
+<utu:ResponsiveView>
+     <!-- Contents -->
+</utu:ResponsiveView>
+```
+
+### Inheritance 
+Object &#8594; DependencyObject &#8594; UIElement &#8594; FrameworkElement &#8594; Control &#8594; ContentControl
+
+### Constructors
+| Constructor      | Description                                               |
+|------------------|-----------------------------------------------------------|
+| ResponsiveView() | Initializes a new instance of the `ResponsiveView` class. |
+
+### Properties
+| Property          | Type             | Description                                             |
+| ----------------- | ---------------- | ------------------------------------------------------- |
+| NarrowestTemplate | DataTemplate     | Template to be displayed on the narrowest screen size.  |
+| NarrowTemplate    | DataTemplate     | Template to be displayed on a narrow screen size.       |
+| NormalTemplate    | DataTemplate     | Template to be displayed on a normal screen size.       |
+| WideTemplate      | DataTemplate     | Template to be displayed on a wide screen size.         |
+| WidestTemplate    | DataTemplate     | Template to be displayed on the widest screen size.     |
+| ResponsiveLayout  | ResponsiveLayout | Overrides the screen size threshold/breakpoints.        |
```

Comment:
This property can be renamed to `Layout` to match the ResponsiveExtension properties

Reply:
i think there was a name clash with one of method of the same name, which is why we added the Responsive- prefix